main.tf

```diff
@@ -1,13 +1,28 @@
-resource "random_integer" "example" {
+# -----------------------------------------------------------------------------------------------------------------------
+# Subscribe the Acccount to Security Hub
+# -----------------------------------------------------------------------------------------------------------------------
+resource "aws_securityhub_account" "enable_us_east_1" {
```

Comment:
why do we use the region in the resource name?
The region is from the AWS provider.
Do we want to use any region?